.github/workflows/get-gke-credentials-it.yml

```diff
@@ -0,0 +1,121 @@
+name: get-gke-credentials Integration
+
+on:
+  push:
+    paths:
+    - 'get-gke-credentials/**'
+    - '.github/workflows/get-gke-credentials**'
+  pull_request:
```

Comment:
For consistency other actions aren't running IT on PRs

Reply:
done